demo/src/app/components/carousel/carousel.component.ts

```diff
@@ -8,6 +8,7 @@ import {DEMO_SNIPPETS} from './demos';
       <ngbd-api-docs directive="NgbCarousel"></ngbd-api-docs>
       <ngbd-api-docs directive="NgbSlide"></ngbd-api-docs>
       <ngbd-api-docs-config type="NgbCarouselConfig"></ngbd-api-docs-config>
+      <ngbd-api-docs-class type="NgbSlideCarouselEvent"></ngbd-api-docs-class>
```

Comment:
Documentation is generated automatically from the source code - as long as you've got JS doc on the `@Output` it will make it to the demo site.